### PR TITLE
[GEOS-7840] Consistently use java-vector-tile:1.0.9

### PR DESCRIPTION
**Pull request by tbarsballe**
See https://osgeo-org.atlassian.net/browse/GEOS-7840

This one will need to be backported to 2.10.x and 2.9.x

### CODE REVIEW

File: src/community/script/groovy/pom.xml [hunk 1]
```diff
@@ -27,10 +33,10 @@
    <version>1.7-SNAPSHOT</version>
   </dependency>
   <dependency>
-    <!-- this dependency taken from geoscript-groovy pom but upgraded to 1.0.8 -->
+    <!-- this dependency taken from geoscript-groovy pom but upgraded to 1.0.9 -->
     <groupId>no.ecc.vectortile</groupId>
     <artifactId>java-vector-tile</artifactId>
-    <version>1.0.8</version>
+    <version>1.0.9</version>
```
Comment (jodygarnett) [inline src/community/script/groovy/pom.xml]: should this be in a dependency management section?

Comment (tbarsballe) [inline src/community/script/groovy/pom.xml]: Based on https://github.com/geoserver/geoserver/blob/master/src/extension/vectortiles/pom.xml#L17 , apparently not?

**Comment by tbarsballe** [comment]
Moved version information to dependency management.

**Comment by tbarsballe** [comment]
Backported to 2.10.x and 2.9.x

**Pull request merged**